src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/ManageApprentices/Index.cshtml

```diff
@@ -67,6 +68,30 @@ else
     </table>
 }
 
+@if (Model.ShowPageLinks)
+{
+    <nav role="navigation" aria-label="Pagination">
+        <div class="das-pagination__summary">
+            Showing @Model.FilterModel.PagedRecordsFrom to @Model.FilterModel.PagedRecordsTo of @Model.FilterModel.TotalNumberOfApprenticeshipsFound
+            reservation@(Model.FilterModel.TotalNumberOfApprenticeshipsFound != 1 ? "s" : null), @Model.FilterModel.TotalNumberOfApprenticeshipsWithAlertsFound with alerts
+        </div>
+        <ul class="das-pagination">
+            @foreach (var pageLink in Model.FilterModel.PageLinks)
+            {
+                <li class="das-pagination__item">
+                    <a class="das-pagination__link @(pageLink.IsCurrent.HasValue && pageLink.IsCurrent.Value ? "current":"")"
+                       asp-route="@RouteNames.ManageApprentices"
+                       asp-all-route-data="@pageLink.RouteData"
+                       aria-current="@pageLink.IsCurrent"
+                       aria-label="@pageLink.AriaLabel">
+                        @pageLink.Label
+                    </a>
+                </li>
+            }
+        </ul>
+    </nav>
+}
+
 @section breadcrumb {
     <a href="javascript:history.back()" class="govuk-back-link">Back</a>
```

Comment:
Not strictly in this PR, but we don't do javascript back

Reply:
Agreed. Doesn't work without javascript

Reply:
what about for providers? can't we assume providers always have javascript.

Reply:
we were told that we could use javascript back for this - if not we'll need to create a new story to cover this

Reply:
It’s ok to use JavaScript back links, we just need to slightly change the way we do it. JavaScript needs to insert the link into the page, so if JS is disabled then users won’t see the link. Not a major issue at all

Reply:
@jamesmking ok, is this something you would do or did you want me to change this? not sure if you have a specific way to do it.

Reply:
Yeah I can do this

Reply:
Spoke with James and this change is deemed out of scope for this ticket and will be handled in a separate story

Reply:
Created ticket MF-425 on Manage Funding backlog that will be used to address this issue.